src/Management/src/Endpoint/Services/ServiceRegistration.cs

```diff
@@ -0,0 +1,82 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the Apache 2.0 License.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Immutable;
+using System.Reflection;
+using System.Text.Json.Serialization;
+using Microsoft.Extensions.DependencyInjection;
+using Steeltoe.Common;
+
+namespace Steeltoe.Management.Endpoint.Services;
+
+public sealed class ServiceRegistration
+{
+    [JsonIgnore]
+    internal string Name { get; }
+
+    [JsonPropertyName("scope")]
+    public string Scope { get; }
+
+    [JsonPropertyName("type")]
+    public string Type { get; }
+
+    [JsonPropertyName("resource")]
+    public string AssemblyName { get; }
+
+    [JsonPropertyName("dependencies")]
+    public ISet<string> Dependencies { get; }
+
+    public ServiceRegistration(ServiceDescriptor descriptor)
+    {
+        ArgumentGuard.NotNull(descriptor);
+
+        Name = descriptor.ServiceType.Name;
+        Scope = descriptor.Lifetime.ToString();
+        Type = descriptor.ImplementationInstance?.GetType().ToString() ?? descriptor.ServiceType.ToString();
+        AssemblyName = descriptor.ServiceType.AssemblyQualifiedName!;
```

Comment:
These values may be incorrect, compared to how Spring works. See related conversation at https://github.com/SteeltoeOSS/Steeltoe/pull/1129#discussion_r1377424010.